src/utils/parseBehaviors.js

```diff
@@ -0,0 +1,37 @@
+import _                        from 'underscore';
+import _getValue                from '../utils/_getValue';
+
+// Takes care of getting the behavior class
+// given options and a key.
+// If a user passes in options.behaviorClass
+// default to using that.
+// If a user passes in a Behavior Class directly, use that
+// Otherwise delegate the lookup to the users `behaviorsLookup` implementation.
+function getBehaviorClass(options, key) {
+  if (options.behaviorClass) {
+    return options.behaviorClass;
+    //treat functions as a Behavior constructor
+  } else if (_.isFunction(options)) {
+    return options;
+  }
+
+  // behaviorsLookup can be either a flat object or a method
+  return _getValue(Marionette.Behaviors.behaviorsLookup, this, [options, key])[key];
```

Comment:
Ah well that's ok..  in that case this still needs to have an import Marionette from 'backbone.marionette';